entries/g/glovoapp.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Glovo": {
+    "domain": "glovoapp.com",
+    "url": "https://glovoapp.com/",
+    "contact": {
+      "facebook": "glovoappES",
+      "twitter": "glovo"
+    },  
+    "categories": [
+      "food"
+    ]

```

Since Glovo only serves certain countries, a regions element would be good.
```suggestion
    ],
    "regions": [
      "ad",
      "am",
      "ba",
      "bg",
      "ci",
      "hr",
      "ge",
      "gh",
      "it",
      "kz",
      "ke",
      "kg",
      "md",
      "me",
      "ma",
      "ng",
      "pl",
      "pt",
      "ro",
      "rs",
      "es",
      "tn",
      "ua"
    ]
```